### ISSUE TITLE
Disabling types takes multiple attemts to work

### ISSUE THREAD

**Issue by DarkReaper231** [KaptainWutax/SeedCracker#149]
Basically when you do /seed finder type EMERALD_ORE OFF for example it doesn't actually turn it off until you do it multiple times and kinda spam that command as well as /seed data clear After few attemts it works but it should work first time.

**Comment by Bluedragonplayz2** [comment]
for my case it really doesnt turn off

**Comment by KaptainWutax** [comment]
Never had this problem before. Is your game lagging?

**Comment by MaHl111** [comment]
Yeah, I have a same problem. No idea what caused it. Disabling Emeralds for example is essential, due to Paper having Anti-Xray setting that duplicates ores to prevent X-raying. Sometimes it really keeps trying to find EMERALD_ORE even when I turned it off and it says it's set to FALSE. And yes, I also cleared data after disabling. For some weird reason, found structures like Desert_Well are not found again after clearing data or disabling mod through command. And sometimes it skips Shipwrecks, even when they are ON, also on server.

**Comment by MrAppu** [comment]
As MaHl111 says, I too had difficulty that the program does not finding desert temples

**Comment by zykrahgaming** [comment]
I modified the mod in a way that makes using it on anti-xray enabled servers a lot easier: https://github.com/zykrahgaming/SeedCracker/releases/tag/v0.2.4-beta-modified
(It just disables emerald ore finder and enables dungeon finder by default)
